bin/start_aws.sh

```diff
@@ -152,8 +152,8 @@ else
 fi
 
 function createpw() {
-  base64 /dev/random | tr -dc '[:alnum:]' | head -c${1:-16}
-  return 0
+    PWORD=$(dd if=/dev/urandom count=1  2>/dev/null | base64 | head -c16)
+    echo $PWORD
```

Comment:
Two questions:

1. So the fix to get around calling `/dev/random` which was causing jenkins to hang was to read it in using `dd`?  My understanding here is that this is just a different way to read from `/dev/urandom` to bypass a jenkins quirk.  Is that correct?  Just trying to understand how I might make a choice between these two approaches in the future.
2. What purpose is `PWORD` serving here?  Would `echo $(dd if=/dev/urandom count=1  2>/dev/null | base64 | head -c16)` not serve the same purpose?